main.js

```diff
@@ -125,19 +137,27 @@ function persistOnPageLoad() {
   }
 }
 
-
 function activeSearchFilter() {
   var titleArray = document.querySelectorAll('.idea-title');
   var bodyArray = document.querySelectorAll('.idea-body');
   var titleCharacters;
   var bodyCharacters;
-  for(i = 0; i < titleArray.length; i++) {
+  for (i = 0; i < titleArray.length; i++) {
     titleCharacters = titleArray[i].innerText.toLowerCase();
     bodyCharacters = bodyArray[i].innerText.toLowerCase();
-      if(titleCharacters.includes(event.target.value.toLowerCase()) || bodyCharacters.includes(event.target.value.toLowerCase())) {
-        titleArray[i].parentNode.parentNode.classList.remove('hidden');
-      } else {
-          titleArray[i].parentNode.parentNode.classList.add('hidden');
-        }
+    if (titleCharacters.includes(searchBox.value.toLowerCase()) || bodyCharacters.includes(searchBox.value.toLowerCase())) {
+      titleArray[i].parentNode.parentNode.classList.remove('hidden');
```

Comment:
Since the event.target was clicking on the searchBox, I just replaced this with searchBox.